### ISSUE TITLE
Review and reassess warning messages

### ISSUE THREAD

**Issue by viniarck** [kytos-ng/sdntrace#41]
Every trace is being logged as warning, should this be kept as warning? Looks like the intention was to signal to network operators that packet-in/outs can impact but if the volume is low should this still be warning? It seems like a debug message. 

```
2023-02-28 12:25:12,444 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Starting Trace Path ID: 30001
2023-02-28 12:25:12,471 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:12,972 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:13,474 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:13,976 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Trace Completed!
```

**Comment by jab1982** [comment]
This was for troubleshooting during the development.  Jeronimo BezerraOn Feb 28, 2023, at 11:12 AM, Vinicius Arcanjo ***@***.***> wrote:﻿
Every trace is being logged as warning, should this be kept as warning? Looks like the intention was to signal to network operators that packet-in/outs can impact but if the volume is low should this still be warning? It seems like a debug message.
2023-02-28 12:25:12,444 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Starting Trace Path ID: 30001
2023-02-28 12:25:12,471 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:12,972 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:13,474 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 
2023-02-28 12:25:13,976 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Trace Completed!

**Comment by viniarck** [comment]
> This was for troubleshooting during the development.  Jeronimo BezerraOn Feb 28, 2023, at 11:12 AM, Vinicius Arcanjo ***@***.***> wrote:﻿ Every trace is being logged as warning, should this be kept as warning? Looks like the intention was to signal to network operators that packet-in/outs can impact but if the volume is low should this still be warning? It seems like a debug message. 2023-02-28 12:25:12,444 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Starting Trace Path ID: 30001 2023-02-28 12:25:12,471 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 2023-02-28 12:25:12,972 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 2023-02-28 12:25:13,474 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Sending POut to switch: 00:00:00:00:00:00:00:01 and in_port 1 2023-02-28 12:25:13,976 - WARNING [kytos.napps.amlight/sdntrace] (Dummy-127) Trace 30001: Trace Completed! —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Right. Thanks for confirming, Jeronimo. I suppose we can set them as debug then.
